src/tools/genpolicy/src/pod.rs

```diff
@@ -896,6 +906,17 @@ impl Container {
         }
         compress_default_capabilities(capabilities, defaults);
     }
+
+    pub fn get_process_fields(&self, process: &mut policy::KataProcess) {
+        if let Some(context) = &self.securityContext {
+            if let Some(uid) = context.runAsUser {
+                process.User.UID = uid.try_into().unwrap();
+            }
+            if let Some(allow) = context.allowPrivilegeEscalation {
+                process.NoNewPrivileges = !allow
```

Comment:
Does this compile with the missing semicolon? I'm guessing Rust considers this as a void expression?